web/client/utils/MapInfoUtils.js

```diff
@@ -152,12 +152,16 @@ const MapInfoUtils = {
         const center = {x: lngCorrected, y: props.point.latlng.lat};
         let centerProjected = CoordinatesUtils.reproject(center, 'EPSG:4326', props.map.projection);
         let bounds = MapInfoUtils.getProjectedBBox(centerProjected, resolution, rotation, size, null);
+        let queryLayers = layer.name;
+        if (layer.queryable) {
```

Comment:
Actually I just noticed when grepping for queryable: there already are traces of a `queryable` field in the layer object [1], but I don't see other traces of it. Should I rename it also?

[1] https://github.com/geosolutions-it/MapStore2/blob/master/web/client/utils/MapInfoUtils.js#L223